internal/service/cloudbackupschedule/resource_cloud_backup_schedule_test.go

```diff
@@ -150,24 +149,21 @@ func TestAccBackupRSCloudBackupSchedule_basic(t *testing.T) {
 }
 
 func TestAccBackupRSCloudBackupSchedule_export(t *testing.T) {
-	acc.SkipTestForCI(t) // needs AWS configuration
-
 	var (
-		clusterInfo  = acc.GetClusterInfo(t, &acc.ClusterRequest{CloudBackup: true})
-		policyName   = acc.RandomName()
-		roleName     = acc.RandomName()
-		awsAccessKey = os.Getenv("AWS_ACCESS_KEY_ID")
-		awsSecretKey = os.Getenv("AWS_SECRET_ACCESS_KEY")
-		region       = os.Getenv("AWS_REGION")
```

Comment:
I like that we removed all the provider configuration 👍

Reply:
😃